src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -743,7 +743,10 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
             $fs->dumpFile($dir.$file, $code);
             @chmod($dir.$file, 0666 & ~umask());
         }
-        @unlink(\dirname($dir.$file).'.legacy');
+        $legacyFile = \dirname($dir.$file).'.legacy';
+        if (\file_exists($legacyFile)) {
```

Comment:
please remove the `\` here (but keep it for dirname)

Reply:
absolutely, I've found a mix of both in the codebase so didn't know which was the preferred one. all done now